inventory/sample.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -16,26 +16,64 @@ env_id: "<REPLACE WITH VALID ENV ID - i.e: env1>"
 # Note that each AWS region may utilize a different AMI for the same image.
 aws_image_name: <REPLACE WITH VALID AMI>
 
-# AWS instance flavors (i.e.: instance sizes)
-master_flavor: m4.xlarge
-infra_node_flavor: i3.xlarge
-app_node_flavor: m4.xlarge
-cns_node_flavor: i3.xlarge
-
-# Number of master instances to deploy.
-# This is a non-HA sample nventory so don't change this value
-aws_num_masters: 1
-
-# Number of instances to be used as infra nodes.
-# This is a non-HA sample inventory so don't change this value
-aws_num_infra_nodes: 1
-
-# Number of instances to be used as compute/app nodes
-aws_num_app_nodes: 3
-
-# Number of instances to be used as CNS (Container Native Storage) nodes
-aws_num_cns_nodes: 0
-
+# Define custom Master(s) API adn Console Port - Default 8443
+#openshift_master_api_port: 443
+#openshift_master_console_port: 443
+
+# Define custom console and apps public DNS Name
+openshift_master_default_subdomain: "apps.{{ env_id }}.{{ dns_domain }}"
+openshift_master_cluster_hostname: "console.internal.{{ env_id }}.{{ dns_domain }}"
+openshift_master_cluster_public_hostname: "console.{{ env_id }}.{{ dns_domain }}"
+
+# Define infrastructure skeleton
+cloud_infrastructure:
+   region: <REPLACE WITH VALID REGION>
+   image_name: <REPLACE WITH VALID AMI>
+   masters:
+     count: <REPLACE WITH NUMBER OF INSTANCES TO CREATE>
+     flavor: <REPLACE WITH DESIRED FLAVOR FOR THE INSTANCE>
+     zones:
+     - <REPLACE WITH THE AZ(s) LIST TO PLACE THE INSTANCES IN>
+     - <REPLACE WITH THE AZ(s) LIST TO PLACE THE INSTANCES IN>
+     - <REPLACE WITH THE AZ(s) LIST TO PLACE THE INSTANCES IN>
+     name_prefix: <REPLACE WITH DESIRED NAME PREFIX FOR THE INSTANCE>
+     docker_volume_size: <REPLACE WITH THE SIZE (Gi) FOR DOCKER POOL DEVICE>
```

Comment:
This is also defined down below (old variable named `docker_volume_size` ) - old variable should be removed

Reply:
likewise, the docker device should be defined here

Reply:
Re: docker device name we need to figure out how to support this as `pre-install` playbook is expecting just one var:

```# Ensure the 'docker_storage_block_device' is correctly populated for all nodes
- hosts: nodes
  tasks:
    - set_fact:
        docker_storage_block_device: "{{ hostvars['localhost'].docker_storage_block_device }}"```